libs/react-helpers/src/lib/grid-cells/price-cell.tsx

```diff
@@ -17,25 +17,24 @@ export const PriceCell = ({
   decimalPlaces = 0,
   value,
   className = '',
-  invalid = false
+  invalid = false,
 }: IPriceCellProps) => {
-  if (!value || isNaN(Number(value)) || invalid) return <span>-</span>
-  const formattedNumber = getNumberFormat(decimalPlaces).format(value)
+  if (!value || isNaN(Number(value)) || invalid) return <span>-</span>;
```

Comment:
minor (non-blocking): edge case here of `value = 0` will return "-" when I think we want it to return a formatted price.

Reply:
Good point, I will add it to post merge fix list